app/controllers/admin/topics_controller.rb

```diff
@@ -1,17 +1,20 @@
 class Admin::TopicsController < ApplicationController
-before_action :authenticate_user!
-before_action :check_admin
-
+  before_action :authenticate_user!
+  before_action :check_admin
+  add_breadcrumb "admin", :admin_users_path
+  add_breadcrumb "topics", :admin_topics_path
   def index
     @topics = Topic.paginate(page: params[:page], per_page: 8)
   end
 
   def new
     @topic = Topic.new
+    add_breadcrumb "new_topic", new_admin_topic_path
   end
 
   def show
     @topic = Topic.find(params[:id])
+    add_breadcrumb "show_topic_"+params[:id], admin_topic_path(@topic) 
     @questions = Question.where(topic_id: params[:id]).paginate(page: params[:page], :per_page => 10)
```

Comment:
@vvs79 - imagine that you don't know about Orest's code) - please correct mistake